src/Controls/src/Core.Design/toolbox/Microsoft.Maui.toolbox.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<Catalog Version="3.3">
```

Comment:
This file looks new, where did it come from?

Reply:
I didn't delete the old one.. I initially thought it might be useful to keep the old one for control gallery or something, building on old TFM's because this is what the designer uses to map some things out, but we're changing control gallery namespaces and such too anyway so it's probably not particularly useful to keep the old one.  Fixed in another commit.